2023/index.html

```diff
@@ -85,19 +127,61 @@ <h1>Gonzaga Hackathon</h1>
                     </label>
                 </div>
             </li>
-            <div class="carousel-dots">
-                <label for="content-1" class="carousel-dot" id="content-dot-1"></label>
-                <label for="content-2" class="carousel-dot" id="content-dot-2"></label>
-                <label for="content-3" class="carousel-dot" id="content-dot-3"></label>
-                <label for="content-4" class="carousel-dot" id="content-dot-4"></label>
-            </div>
         </ul>
     </section>
+</div>
 
+<div class="carousel-dots">

```

Comment:
this was in the <ul>, what is the purpose of moving it out?

Reply:
In the what?

Reply:
what is this saying?

Reply:
I am wondering why the dots at the bottom of the carousel were moved outside of the list element. They were previously a part of the list

Reply:
They were disappearing/going in the wrong spot since I added another div, and this was my solution.

Reply:
ok, thanks